spec/java_buildpack/framework/jrebel_agent_spec.rb

```diff
@@ -21,35 +21,56 @@
 describe JavaBuildpack::Framework::JrebelAgent do
   include_context 'component_helper'
 
-  it 'does not detect without JRebel config files present' do
-    expect(component.detect).to be_nil
-  end
+  context 'detect' do
```

Comment:
We don't generally add additional context for the phases, so let's remove this one (and `compile` and `release`) to better match the existing codebase.